app/javascript/spotlight/admin/edit_in_place.js

```diff
@@ -43,5 +43,24 @@ export default class {
         return false;
       });
     })
+
+    $("[data-behavior='restore-default']").each(function(){
+      var hidden = $("[data-default-value]", $(this));
+      var value = $($("[data-in-place-edit-target]", $(this)).data('in-place-edit-target'), $(this));
+      var button = $("[data-restore-default]", $(this));
+      hidden.on('blur', function(){
```

Comment:
@marlo-longley I've been looking into the test failure you and @corylown were asking about. Two things:
1. I think maybe we need to handle more than blur, depending on the behavior we are looking for.

* If I click "Language", type "Language of Origin" and press Enter, I do not get the "Restore default" button in the current page
*  If I click "Language", type "Language of Origin" and invoke blur by clicking somewhere else (including the save changes button) I immediately get the "Restore default" button in the current page


2. Right now, there is an immediate check for that restore default button as an indication that the save has occurred. I don't think that's going to be reliable as we don't know for sure if that find will happen before or after the page reload.
https://github.com/projectblacklight/spotlight/blob/6fbece146028df810248766d7d342a2f180201ec/spec/features/exhibits/edit_metadata_fields_spec.rb#L41-L43

If we want the "Restore button" to show immediately, could we cheese it a bit for the test and force a page revisit? Add sleep before the config data check?

Reply:
Group decision: skip this test.